src/utilities/configuration.ts

```diff
@@ -57,11 +52,16 @@ if (!payerMnemonic) {
   throw new ConfigurationError('SECRET_PAYER_MNEMONIC is not provided');
 }
 
-const w3nOrigins: Record<string, string> = {
-  'wss://peregrine.kilt.io': 'https://test.w3n.id',
-  'wss://peregrine-stg.kilt.io/para': 'https://smoke.w3n.id',
-  'wss://kilt-rpc.dwellir.com': 'https://w3n.id',
-};
+function deductW3nOrigin(blockchainEndpoint: string) {
+  const endpoint = blockchainEndpoint.toLocaleLowerCase();
```

Comment:
```suggestion
  const endpoint = blockchainEndpoint.toLowerCase();
```
I believe locale is irrelevant here

Reply:
Yes, it was autocomplete.

Reply:
done